doc/source/io.rst

```diff
@@ -4428,16 +4428,11 @@ DataFrame with a shape and data types derived from the source table.
 Additionally, DataFrames can be inserted into new BigQuery tables or appended
 to existing tables.
 
-You will need to install some additional dependencies:
-
```

Comment:
you need to add a pointer from here to the Dependency section of the docs (that you added below).